_posts/2016/2016-03-24-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+
+{% image blog/blog-ecommerce-barcamp-1.jpg %}{% endimage %}
+
+## The setting
+
+The event began with a dinner on Thursday evening in the famous local restaurant [Zur Noll](http://www.zur-noll.de/), which should break the ice of everyone's comfort zone and kickoff the communication flow.
+
+On Friday morning all participants met in the auditorium at the [Ernst-Abbe-Hochschule Jena](http://www.eah-jena.de/), University of Applied Sciences, which served as hosting space. After a collective breakfast the session planning for both days took place. Everybody can suggest an own Open Space talk. The crowd expresses interest in a suggestion via raising their hands. This immediate reaction allows a proper planning of the time slot and room allocation for the session.
+
+## The sessions
+
+To give you a better impression of the actual contents of the conference I want to outline some of the sessions. As I am a backend software craftsman you may recognise easily that I skipped the marketing and lawfully oriented parts of the event and tried to focus on the more technical talks.
+
+{% image blog/blog-ecommerce-barcamp-2.jpg %}{% endimage %}
+
+The first session was held by Christoph Rüger, who is part of a small team that has built a new cloud service that reads data from files (uploaded CSV, XML, Google Drive etc.) and let's the user create workflows via Drag'n'Drop (e.g. cronjobs with conditions and scheduling) that can interact with other service providers. Christopher revealed which tools his team used for internal project management and customer communication. He also explained the process of finding the fitting frameworks for developing their service, which was a quite interesting to follow.
+
+The next presentation dealt with the topic "Infrastructure as Code". Jan Peschke and Manuel Dieckmann prepared a talk about how they automated the rollout and development of Shopware environments with [Ansible](https://www.ansible.com/) and [OpenStack](https://www.openstack.org), a solution similar to [Amazon AWS](https://aws.amazon.com). Currently this cloud technology is on the transition to the state of production-readyness but not there yet. However a lot of minor service providers already emphasize OpenStack in the marketing ads as a blazing part of their customer portfolio. In the session it was made clear that developers have to be very careful with such announced offers as the promises of most providers can be very disappointing and do not hold up to an usable support of OpenStack. So be careful, if you opt against rackspace. Nevertheless, the framework in general is very interesting and already provides a lot of functionality. It will be worth to follow in the future.
```

Comment:
kurz auf deutsch: Also die meisten Hoster bieten schon OpenStack in ihrem Portfolio an, jedoch ist das mit Vorsicht zu genießen, da es in 99% der Fälle nicht sauber untersützt wird. Meist gibt es Probleme beim Networking. Als einer der wenigen Hoster bietet bisher nur rackspace ordentlichen support. Dies hat die interne Evaluierung seitens der Speaker gezeigt. Allerdings muss man sagen das Jan Peschke von SysEleven, also einem speziallisiertem Hoster für Shops und  er damit in Konkurrenz zu den großen Hostern wie Hetzner, HostEurope etc. pp. steht. O-Ton wir haben es schon drauf.